public/app/features/dashboard/containers/DashboardPage.tsx

```diff
@@ -269,10 +282,10 @@ export class DashboardPage extends PureComponent<Props, State> {
     locationService.push('/');
   };
 
-  renderSlowInitState() {
+  renderSlowInitState(dashboardLoadingstyle: string, dashboardLoadingTextStyle: string) {
```

Comment:
Maybe we should pass the styles object into the render functions instead? Or even better take this opportunity to refactor out smaller components from these render functions. WDYT?

Reply:
I think passing the styles object will do in this case. I don't think these render functions are worth refactoring as a standalone component because it's not really reused across other components.

Reply:
Do you prefer bigger complex classes over smaller functional components? I don't and it's not reuse I was thinking about in this case.

Reply:
Oh...I get your point now. In that case, then it makes alot of sense to refactor the render function.